### DISCUSSION TITLE
Doubts regarding Application

### DISCUSSION THREAD

**Discussion by Rohan-Makwana** [github/india#626] [Students]
It's more than 4 days since I had applied, but my application is still under review. While applying, GitHub externship asked for proof that I'm a student either by verifying my college email address or submitting some proof that I am in college. I went for the latter since my college blocks emails from external organizations. So can this be the reason that my application is still under review? Are others facing similar issues?

**Comment by crystal-kishore** [comment]
No bro till now  its under review , for safer side If you check your dashboard there will profile option go and update your profile by uploading your clg id or marksheet.

**Comment by Cyborg2900** [comment]
I am facing the same issue. I was not able to verify my college email because I did not receive the OTP mail. I guess my college too blocked emails from external organizations. I went for the marksheet and college ID proof but it has been 4 days and still it shows under review.

**Comment by punitjain03** [comment]
Hi  , Currently we are reviewing the profiles, From today EOD or tomorrow morning, We will notify the status of the application status. Post that you can apply for the organisations.

**Reply by Rohan-Makwana** [reply]
Thank you for the reply. I hope my application gets accepted.

**Reply by santanukumar666** [reply]
Hey My profile is still under review

**Reply by skmodi649** [reply]
My profile is still under review, please reply!
